client/.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_API_ACCESS_KEY = AKIA3ENAIKTK2KB2V5HQ,
```

Comment:
if you're going to use .env ~ lets add it to our .gitignore
remember we dont want these keys anywhere in github